adaptivefiltering/pdal.py

```diff
@@ -15,6 +22,7 @@
 import pdal
 import pyrsistent
 import tempfile
+import richdem as rd
```

Comment:
I am not a huge fan of these abbreviations unless its `np` which everybody knows.